frontend/packages/console-plugin-sdk/src/typings/storage-class-params.ts

```diff
@@ -32,7 +32,7 @@ export type ExtensionSCProvisionerProp = {
           name: string;
           hintText: string;
           value?: string;
-          visible?: () => boolean;
+          visible?: (params?: any) => boolean;
```

Comment:
@spadgett PTAL.

Reply:
needs your approval on this.

Reply:
Can you point me to the place in the code where the parameters are being passed? Can we give this a more specific type?

Reply:
this is where we are calling this: https://github.com/openshift/console/blob/10788fddfaaed31f0b5b64d34690d5a5f324652a/frontend/public/components/storage-class-form.tsx#L863

Reply:
https://github.com/openshift/console/blob/10788fddfaaed31f0b5b64d34690d5a5f324652a/frontend/public/components/storage-class-form.tsx#L1162

Reply:
OK, so we were already passing the parameters, but the type declaration was simply wrong?

Reply:
Yes.